### PR TITLE
Add some getting started docs

### PR DESCRIPTION
**Pull request by JLLeitschuh**
This is just some of the basics that really threw me when starting to write my build logic in kotlin.

### CODE REVIEW

File: doc/getting-started/Closures.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Groovy Closures in Gradle Script Kotlin
+
+Most third party plugins are written in groovy with the expectation that closures will 
+be passed as arguments. In order to provide a way to construct closures while preserving kotlin's
+strong typing two helper methods exist: `closureOf<...> {...}` and `delegateClosureOf<...> {...}`.
```
Comment (JLLeitschuh) [inline doc/getting-started/Closures.md]: Perhaps these should be in a list?

File: doc/getting-started/Configuring Plugins.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        ruleSetFiles = files(new File(rootDir, "pmd-ruleset.xml"))
+        ruleSets = []
+    }
+
+    findbugs {
+        sourceSets = [sourceSets.main, sourceSets.test]
+        excludeFilter = new File(rootDir, "findBugsSuppressions.xml")
+        effort = "max"
+    }
+```
```
Comment (JLLeitschuh) [inline doc/getting-started/Configuring Plugins.md]: I may want un-indent these.

**Comment by JLLeitschuh** [comment]
This good to go?

**Comment by bamboo** [comment]
Hi @JLLeitschuh, I'll be integrating that soon, as part of the next milestone. 

In the meantime, could you please sign the [Gradle CLA](https://gradle.org/contributor-license-agreement/)?

Thank you very much for you contributions so far!

**Comment by JLLeitschuh** [comment]
Waiting on my manager to sign it.

**Comment by JLLeitschuh** [comment]
My manager signed the CLA. This should be good to go.

**Comment by bamboo** [comment]
Thank you, @JLLeitschuh!

**Comment by JLLeitschuh** [comment]
Closed instead of being merged??

**Comment by bamboo** [comment]
Oops...

**Pull request merged**